src/Workspaces/Core/Portable/Workspace/Solution/ProjectState.cs

```diff
@@ -239,7 +234,7 @@ private static async Task<VersionStamp> ComputeLatestDocumentTopLevelChangeVersi
 
         internal DocumentState CreateDocument(DocumentInfo documentInfo, ParseOptions? parseOptions, LoadTextOptions loadTextOptions)
         {
-            var doc = new DocumentState(documentInfo, parseOptions, loadTextOptions, _languageServices, _solutionServices);
+            var doc = new DocumentState(LanguageServices, documentInfo, parseOptions, loadTextOptions);
```

Comment:
Kinda wish language services was towards the end of the parameter list -- it's the least important of all the parameters, kinda...